app/cdap/components/WranglerV2/SectionWrapper/index.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+import { grey } from '@material-ui/core/colors';
+import React from 'react';
+import SectionComponent from 'components/WranglerV2/SectionComponent';
+import styled from 'styled-components';
+import T from 'i18n-react';
+import { Typography, Button, Divider } from '@material-ui/core';
+import { IHeaderNamesList } from 'components/WranglerGrid/SelectColumnPanel/types';
+
+export interface ISectionWrapperComponentProps {
```

Comment:
do not use the work component in naming the component or its props or values unless it adds more meaning

Reply:
changed ISectionWrapperComponentProps to ISectionWrapperProps